test/index.js

```diff
@@ -134,18 +126,27 @@ describe('interpret.extensions', function() {
         }
       }
 
+      // Any swc test after the first will fail on linux due to https://github.com/swc-project/swc/issues/4107
```

Comment:
Every test for swc after the first fails due to https://github.com/swc-project/swc/issues/4107